image/templates/helm/shared/templates/02-scanner-v4-07-indexer-deployment.yaml

```diff
@@ -61,12 +61,18 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-       {{- if ._rox.env.openshift }}
+        {{- if ._rox.env.openshift }}
         - name: ROX_OPENSHIFT_API
           value: "true"
         {{- end }}
         - name: ROX_METRICS_PORT
           value: ":{{- ._rox.scannerV4.indexer.metricsPort -}}"
+        {{- if ._rox.monitoring.openshift.enabled }}
+        - name: ROX_ENABLE_SECURE_METRICS
+          value: "true"
+        - name: ROX_SECURE_METRICS_PORT
+          value: ":9091"
```

Comment:
nit: won't block the PR over this, but I think we should make this configurable like we do for the insecure metrics port. Something like `":{{- ._rox.scannerV4.indexer.secureMetricsPort -}}"`

Reply:
This would be customer-facing, right? I don't see a need for that at the moment, and I prefer to leave this out to avoid adding new values and increasing the scope of this change.